api/src/main/java/io/onetable/model/stat/Range.java

```diff
@@ -63,44 +56,23 @@
  */
 @Value
 public class Range {
-  RangeType rangeType;
+  // define a reusable range for null values in case of sparse data
+  private static final Range NULL_RANGE = new Range(null, null);
```

Comment:
Optimization for sparse data. We would otherwise create lots of "null" ranges for files that did not have any values for a column. Now we will just reuse this existing object.